app/models/diff.js

```diff
@@ -4,6 +4,9 @@ var comment = /^\\/,
     deleted = /^\-/,
     inserted = /^\+/;
 
+const targetFileBaseUrl =
+  'https://raw.githubusercontent.com/railsdiff/api/master/generated/';
```

Comment:
Also, this value should probably get pulled from the `config/environment.js` file.